l10n_eu_oss/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_oss_tax_rate_user,access_oss_tax_rate_user,model_oss_tax_rate,,1,0,0,0
```

Comment:
I think there's no sense in giving this general read access, isn't it?

Reply:
It makes sense, I change it